client/src/app/s-degree-plan-editor/s-degree-plan-editor.component.html

```diff
@@ -0,0 +1,252 @@
+<div class="outermost-container">
```

Comment:
HTML. It's better to just look at how the page looks rather than reading through this.